### PR TITLE
Fix badges in README.md

### LINKED ISSUE #79: Invalid Badges in README.md
Readme error

The details in the badge template of README.md is incorrect, due to which the 'repo not found' message is being shown on the badges.

### PR DESCRIPTION
**Pull request by namit-chandwani**
### What does it fix?

Closes #79  

I've corrected the repo details in the badge template inside README.md. The badges now point to the correct repo.

### CODE REVIEW

**Review by RaduCStefanescu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project name
```
Comment (surdu) [inline README.md:1]: While you are here, you might also want to set the project name and URL

**Review by surdu — CHANGES_REQUESTED** [review]

**Review by surdu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (namit-chandwani) [reply README.md:1]: Sure

Comment (namit-chandwani) [reply README.md]: Sure!

Comment (namit-chandwani) [reply README.md]: I've updated the project name.
What would you like to set the URL as? Like is the project hosted anywhere?

Comment (surdu) [reply README.md]: The URL should be https://rezultatevot.ro

**Review by surdu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (namit-chandwani) [reply README.md]: Okay

**Review by surdu — APPROVED** [review]
🚀

**Review by Utwo — APPROVED** [review]

**Pull request merged**